airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/RouteToSyncTaskQueueActivityImpl.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling.activities;
+
+import io.airbyte.workers.temporal.sync.RouterService;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class RouteToSyncTaskQueueActivityImpl implements RouteToSyncTaskQueueActivity {
+
+  @Inject
+  private RouterService routerService;
+
+  @Override
+  public RouteToSyncTaskQueueOutput route(final RouteToSyncTaskQueueInput input) {
+    return new RouteToSyncTaskQueueOutput("SOME_TASK_QUEUE");
```

Comment:
@pmossman Is this just a temporary placeholder?  I would assume that we would call the `routerService` here.

Reply:
Ah yep forgot to update this after injecting router service, fixed